drivers/net/ethernet/renesas/rswitch.h

```diff
@@ -216,7 +216,7 @@ struct rswitch_gwca_chain {
 #define RSWITCH_MAX_CTAG_PCP	7
 #define MAX_MTU_SZ		9000
 #define MAX_DESC_SZ		2048
-#define VMQ_INFO_VERSION	1
+#define VMQ_INFO_VERSION	2
```

Comment:
nit: I think it is worth to mention this in commit message ("due to changes in ... needed to bump ...")

Reply:
Thank you for the review. It's a good point - I will add this to commit message.